secboot/secboot_tpm_test.go

```diff
@@ -1082,7 +1079,7 @@ func createMockSnapFile(snapDir, snapPath, snapType string) (snap.Container, err
 }
 
 func mockSbTPMConnection(c *C, tpmErr error) (*sb.TPMConnection, func()) {
-	tcti, err := os.Open("/dev/null")
+	tcti, err := tpm2.OpenTPMDevice("/dev/null")
```

Comment:
I'm surprised that this works, although OpenTPMDevice only checks if the opened file is a character device, which /dev/null is.

As a side-note, I started adding a public API to go-tpm2 to assist with unit testing (https://pkg.go.dev/github.com/canonical/go-tpm2/testutil).